common/cache/class.PsrWrapperCache.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA
+ *
+ */
+
+use oat\oatbox\service\ConfigurableService;
+use Psr\SimpleCache\CacheInterface;
+use oat\oatbox\cache\SimpleCache;
+
+/**
+ * Wrap the PSR simple cache implementation into the legacy interface
+ */
+class common_cache_PsrWrapperCache extends ConfigurableService implements common_cache_Cache
```

Comment:
This class provides the legacy interface and should be discouraged from being used, as is present for backward compatibility only.

Reply:
> should be discouraged from being used
In that case it has to be marked as deprecated.
As we are going to get rid of legacy autoloader it's better to not create new classes in the legacy namespace

Reply:
done

Reply:
Introduce in the PR a new deprecated class and then configure it to be used by default does not seem to me like a good practice! 

How difficult will be to change it's usage to not catch `common_cache_NotFoundException`? I found 14 catches of that exception in three extensions:
tao
taoAct
taoMpArt (that one can be omitted)

I could help with cleaning that up.

Reply:
I removed all references to old cache from generis, so that we can move quicker when it comes to cleaning up old references, but I still don't want to make this feature backward incompatible, even if we could fix all other known extensions quickly